microservices/nris_api/docker-compose.yaml

```diff
@@ -0,0 +1,45 @@
+version: "2"
+
+services:
+  ####################### NRIS_BACKEND Definition #######################
+  nris_backend:
+    restart: always
+    container_name: nris_python
+    build:
+      context: backend
+    ports:
+      - 5500:5500
+    volumes:
+      - ./backend/:/app/
+    depends_on:
+      - nris_database
+    # Export environment variables from .env into the container environment
+    entrypoint: ["bash", "-c", "grep -v '^#' .env | xargs ; flask run"]
+    networks:
+      - nris
+
+  ####################### NRIS_Postgres Definition #######################
+  # Only for Local, openshift will share one db pod for all MDS services
+  nris_database:
+    restart: always
+    container_name: nris_postgres
+    build:
+      context: postgres
+      dockerfile: Dockerfile.dev
+    environment:
+      - POSTGRES_USER=nris
+      - POSTGRES_PASSWORD=VsPJAuMJHu3IVMEv
```

Comment:
Is this just for local? Usually local db's have passwords like password or something easy so you dont have to remember it to copy/paste etc.
Just a personal preference :)

Also im hoping this password isnt used anywhere 😅

Reply:
no it's not just for local, its the same password coded into the DC until we can dynamically create that pw, because this user is create by the same flyway script that runs on openshift, they are the same credentials for now .